React/Base/RCTTVRemoteHandler.m

```diff
@@ -100,9 +110,16 @@ - (void)selectPressed:(UIGestureRecognizer *)r
   [self sendAppleTVEvent:@"select" toView:r.view];
 }
 
-- (void)longPress:(UIGestureRecognizer *)r
+- (void)longPlayPausePressed:(UIGestureRecognizer *)r
+{
+  [self sendAppleTVEvent:@"longPlayPause" toView:r.view];
+  // If shake to show is enabled on device, use long play/pause event to show dev menu
+  [[NSNotificationCenter defaultCenter] postNotificationName:@"RCTShowDevMenuNotification" object:nil];
```

Comment:
Instead of using a string here, can you can change the definition of RCTShowDevMenuNotification to extern (put an RCT_EXTERN definition in the header)